10.11/Dockerfile.rhel10

```diff
@@ -0,0 +1,74 @@
+FROM registry.stage.redhat.io/ubi10/s2i-core
+
+# MariaDB image for OpenShift.
+#
+# Volumes:
+#  * /var/lib/mysql/data - Datastore for MariaDB
+# Environment:
+#  * $MYSQL_USER - Database user name
+#  * $MYSQL_PASSWORD - User's password
+#  * $MYSQL_DATABASE - Name of the database to create
+#  * $MYSQL_ROOT_PASSWORD (Optional) - Password for the 'root' MySQL account
+
+ENV MYSQL_VERSION=10.11 \
+    APP_DATA=/opt/app-root/src \
+    HOME=/var/lib/mysql \
+    NAME=mariadb \
+    VERSION=10.11 \
+    ARCH=x86_64 \
```

Comment:
Is this variable necessary when it's not used anywhere in the code?

Reply:
Good point, I don't know why it is present here. We do not use it in our CI.

Reply:
Can you please remove it, so we don't include a dead code here, otherwise looking at tests LGTM and I'll approve it once it's removed

Reply:
Good point, I don't know why it is present here. We do not use it in our CI.Fixed by https://github.com/sclorg/mariadb-container/pull/262/commits/c815204ab701c89adc069af3aa1b71d74e87ed4d.